### ISSUE TITLE
Does not render SVG in Svelte 5 - requires addition of <svelte:options namespace='svg'/> in Icon.svelte

### ISSUE THREAD

**Issue by TristanBrotherton** [RobBrazier/svelte-awesome#1059]
### Version

3.3.0

### Bundler

sveltekit

### Description

When using Svelte 5 - the SVG is rendered in the DOM as expected, but the browser will not render the SVG. However if you copy the HTML that is in the DOM from the chrome inspector, and render it as a string, the SVG will render as expected.

I have created a [REPL](https://svelte-5-preview.vercel.app/#H4sIAAAAAAAAE4VSwW7bMAz9FY4XX6LEUmzHTuNgG3bpoaftNg2oYiuxNllyJdVdG-TfB9npUAwFdiGpx_coguQZj0pLj9vvZzSil7jFT8OACwzPQ3z4UeogcYHePromIjvfODUE0MKc6iT4ZM8NgOoH6wLcNtbA0dkekllJxJP0tpfJDTc8XFmx9kT9FoP_0g9SundZK9VY41cxPwu0DBAxqCdRxN6gX4NT5gQ13O_8eIJROq-sqTnSJeUIjRbe1xyPgkw1rn_Rvsm78SdHeFJt6CI9W5b5hmbs1ZYcoZPq1IWYLTiCcEoQLQ5S1xw5grNa1hwHJ700QQRlDUcYlXz6bH_XHFNIgRZFBnRTMY7gw_MkOCqtt3DQovl1w3G_G0TooK053hVZCmVVjGRdZh1heTGyvHhIIV_DerPMoUqXeYhmenaUlS938w_rLBtpxTpCac5GQiumKSvJDLHygdC8gpSwDV3mhFK2zMPsZmgka5ZqUmSkyGDNoqTLylRfw6qIIdCKzfmxTNOX2Poq9r7ffSCEkFcH__j33cqPp_193CQ3u9V8fvu42i9WejA2gJOmlW7LzW46wHl6yTS8eXYJN60Ioj7HzV64We25uW2lCaoRGvx8GG0s97fU-WMXev3mcC64wN626qhki9vgHuXlx-UPI5iYWEADAAA=) showing this issue, and additionally created a second REPL of the relevant components and a fix [here](https://svelte-5-preview.vercel.app/#H4sIAAAAAAAAE7UZa3PjtvGv7PHckMxJotS015YS7UuaduIPvd6cO5MPolNDJCihRwEsAD0cWf-9gwfBhx53SSfjGYvEPrC72BeWB68gJRZePD94FK2xF3vfVpU38ORzpV7EFpcSewNPsA3P1MpMZJxU8jalqSTrinEJ9xmjUHC2htQbReptZOhSb9pCW2DMDZpvwEO0w4KtcUQyRkWk4L4mKLEEtQaJJpqmdBY129KZ3jBHEiUHhXaE6NYbeGuWk4Lg3Isl3-DjwGmk0K-rpJdiVknCqABFJiqU4cQX26Uf3aZ09rBdphTgUKIFLo_6cUdyuTKPK0yWK2meF2yvH4R8LnFyyNh6QSjOH9SrwRAVoeapKEllngjdYi4sQrUp7WNWIiGSg_55j9YWPhqNbm44FvIDZ5U4pvRWLc9EyaR-Aji8JoU24fdIoqNZ662OKiRXwsEUFKNs1YMDEqAeWngAM7WixdAgiG5bXCLFpdkyIsWl_Vn5vGT0qggWRUthnnuCmEUji0X4leJwtGvxnn1EO1gQmseNn2lbtri3mLnHWWRPYRY9bJfGXY3rQsaoxHuZpMpTNyVOPSgRXSapJ0Xqaa54ryOFUIl5gTIMf0fKeb_HBaFEOScczH4VxwXZxyAkJ3Q5NYtKRuUkZ5ZjmNdS0816gfkAogi0B59ZN-5cAwyz-aMGlWSJ5IZj0YVq2IaSjOW4C4EXx0DhiO3yA5IrZUqD96ilPCpDndFfaf9BO5vVJb_raZf33nXY9ZGE5OxTb_XqlrVbWWszQqX4_EbXWCqNa37a7rG1uGVnbN5b1CF4FzszzB9rgA2MGmZeHZij3TXJVCbUdP9SDwl8xBnj-aw-ylrcW3g5ccCTXGydu-_JNuerKDIpfxR9RDtbF_xpC-Vhu3QoD9tlC0UhqVLgsh8k4NcAq4qCqwiNnT7TmqwO2bix_wtsaK50wfm0x0RkqFQbTE4AFVGFqEClwH2YzdqXwDqPXwKq3B-Dv2Kc_MyoRKUPL-BvMZckMy9OVkguy60LUn3WtYG6CihHvYhxaJkXCfMCR2vkKDJeTFHpDmNFypxj-oP2V0hgPO2DflTu3YGwjcT8oWXhGtCJBAVraLoB0YV1imrH0RV0wdqZUa0WG5rp9EkZX6OSCKzcIeg6TnjeUeqgVZzpaXY972l16i0geKV2CWsuABzLDae9AwU4AlbOoij83IWbD8R0Oi0GqlliJR5hzhkPmpKVek82pAodtPpITWS9KxiXdbdVoewTWuKhUiZqUJ-AMgliUykWOIchfCgxEhg2AkPN-dDmffwF3P0nIFRIjPLUq0UOp7_AJnVxcxaBr74CvXrORtRkC7U6qimVe3eOzhyeaXMScMlTu32hQtKoBNs_Rm9NP2qsgHPYEbmKW-jOPORLjKNt4xSa_l98GnLlFxLmOqQGNnwGoP6Uio9tcyhT_G2flZscz2r3nxtjPrbyv2OtTuBbztHziAj9GyiWYcvg4Oyofh2hPcEzaHMtVIPotqpDMGlTGaWad6tdj21sfkZrVAVGQkhu22ycpx1U36C72OO0AR9DJ07XAw_tI6KLDSlN0Iuev_1z8R-cydEn_Cx0cgnn40bFlmYKNlc0NdSqb6Uj7SxyvJTCcp1Qg7CToDbrh04hM2enCj4rbI17lSTgu1Dz21HTUL_XWTfQFGFXSMWQiPfofVDjh_Dy0hDPEhifSVc7xGng39MtKkkOFWdVrP9D6uldUg_Eim3KHBYYUJ322RZzGI_8k1zRlJSzJnTSfN3H7NszQ2W2KZHE37F9Y0ytpD2G0-T9NIYx3LhLwchcBdsr9kb4dFY4S-6oOsgX5fuIJGFdCV9dFnFydud_ILkardE-6Io-gJ7gIUQweXtdHl3mu_J0OoBTmTrgM051UZmeuBCdWCWEr08j46wJxte1Mn3NebV-sLa5pJeB_zrFjN1_S810s9RNF3Uf1XSFRmDdMupEQTdl2QlmS_AmMX1lVxDdbpKfVQK5JLJmr1GSBCYt1trQlv2oxHSpesikm0ucqr5_Wjnqw7A8To_BCa8zoK8aiFduR0xz8SORq8Cf-uEFjf0p-OcD-sbNeY6qcxgqDWO4OajfI153Ivsmdr7VVITTxrQ2WNvMopXvdrbL7sdj5z7ZhteebRE6PXQbz_qJRVuwfRuok2StjbsJXpu-fUS7q8O3pb4vHt6t5LoEM4SZRcsvuVzqa-yhadrrVummJIsoY-uKUUylaE8kO9fKDRdMXSzG-_wPf_nmT9NO2CyxvM-baLHIb5q6Wh99gYY3BwMeSfagG8xg8jbspnO1IUe7z95gbdnvi_IR7YIuUrcO6Ib45QX0gxplnaYa3z-NVY52rY5E0TnnUL0kyUXcHw6Y31vVntVNUsPFPo04rkqU4SBKUxHcxft1GYd3JE_mqec_3gXzn1LPD9NUPL4Jo-UAgjWSmSpCebdhM1JsKPnvBt-rJGUPpWmpcjEn-aO-HxukfpvwBCRPUu_mUCMcU88V5dBa-poKrvN7HdzFwfwnP_Ws5C97PRfCPE0DkqdpEMz91Hu8CxVW-PgmTNPfp2mYpqFSsuZT68rR7j4fwL8HYLnc5_1utT4EI9V9rg7YIbfaVtMJaPAra5HwbNert273uyd59en1zcGycL2Lwwu7nm8t1UttxslbLhtOvzBRPGyXn5nS6xG8mrgQRpPUm4wm5jJppuSpV6Chvt205-UNQlygoRrmJM0A3q3rSU3SH7wriB3xJJ0JvYOqIc6wGeEkeqKvy1Z7sHOGpp7ztCnc7Ofa9wXECRrqoU_S-hjBmfrWoN_hDnyyXvoQg19xLDCVSI8SNOKW4N13bJ-4zxQH0XyYuPJdQY-9Z5FwM21FpcEjZ3XjHTkRVYmeYyC0JBQPFyXLPlnPKUhZxiqXckzlX1nJeO09hk_PnDVLyREVBePr2FTAYDgZwCQ8Q1ub8CLlZADDPqWe8FkCRMlamyuGGjIRMBZAqJ7JYFBKoZ7c9RywLhZKsxheF0XRxTMDwatbuX2ExJUI_tyS9d0n_Fxw9XEKelKPf9fEe0tlziSSOBjneNnrGCfj6yTfvD0hMvXeHvvl2twZ2PanjukXzh1bpfF0Vt4Cnhmat6CtOaASQF26xjA-U35_2xFvIwFl1HUgUQTmeyMqRV-cqwPbi0PfK0n28fg_BX7VN-kdAAA=). If you look at the modified `Icon.svelte` I have added `<svelte:options namespace='svg'/>` on line 1. I found the workaround [here](https://github.com/sveltejs/svelte/issues/7613).

### Reproduction URL

https://svelte-5-preview.vercel.app/#H4sIAAAAAAAAE4VSwW7bMAz9FY4XX6LEUmzHTuNgG3bpoaftNg2oYiuxNllyJdVdG-TfB9npUAwFdiGpx_coguQZj0pLj9vvZzSil7jFT8OACwzPQ3z4UeogcYHePromIjvfODUE0MKc6iT4ZM8NgOoH6wLcNtbA0dkekllJxJP0tpfJDTc8XFmx9kT9FoP_0g9SundZK9VY41cxPwu0DBAxqCdRxN6gX4NT5gQ13O_8eIJROq-sqTnSJeUIjRbe1xyPgkw1rn_Rvsm78SdHeFJt6CI9W5b5hmbs1ZYcoZPq1IWYLTiCcEoQLQ5S1xw5grNa1hwHJ700QQRlDUcYlXz6bH_XHFNIgRZFBnRTMY7gw_MkOCqtt3DQovl1w3G_G0TooK053hVZCmVVjGRdZh1heTGyvHhIIV_DerPMoUqXeYhmenaUlS938w_rLBtpxTpCac5GQiumKSvJDLHygdC8gpSwDV3mhFK2zMPsZmgka5ZqUmSkyGDNoqTLylRfw6qIIdCKzfmxTNOX2Poq9r7ffSCEkFcH__j33cqPp_193CQ3u9V8fvu42i9WejA2gJOmlW7LzW46wHl6yTS8eXYJN60Ioj7HzV64We25uW2lCaoRGvx8GG0s97fU-WMXev3mcC64wN626qhki9vgHuXlx-UPI5iYWEADAAA=

### Reproduction steps

```bash
1. View REPL.
2. Inspect REPL and find SVG that is not rendering. 
3. Copy SVG node in inspector
4. Paste HTML into iconStrong
5. SVG will render.
```

**Comment by TristanBrotherton** [comment]
I have pushed a fix to: https://github.com/RobBrazier/svelte-awesome/pull/1060

**Comment by RobBrazier** [comment]
Released this in 3.3.1

**Comment by TristanBrotherton** [comment]
FYI - this issue was also recently fixed in Svelte 5 - which means this could be rolled back if you'd like:
https://github.com/sveltejs/svelte/commit/cd2263fdabd97dcb5e7d99b891580a0757359e3b
